### PR TITLE
Add new page method "remove_rotation"

### PR DESCRIPTION
**Pull request by JorjMcKie**
The new method sets a page's rotation to 0 while keeping the page's appearance and content unchanged. This can significantly simplify inclusion of the page in target pages via Page method "show_pdf_page()".

Other changes are additions / corrections to "changes.txt" and improved wordings for in document.rst and page.rst.

Also contained is a fix for issue 3402 (inability to insert/update fields with inter-field calculations.

### CODE REVIEW

File: changes.txt [hunk 1]
```diff
@@ -11,6 +28,7 @@ Change Log
 
   * **Fixed** `3331 <https://github.com/pymupdf/PyMuPDF/issues/3331>`_: Document.pages() is incorrectly type-hinted
   * **Fixed** `3354 <https://github.com/pymupdf/PyMuPDF/issues/3354>`_: PyMuPDF==1.24.1: AttributeError: property 'metadata' of 'Document' object has no setter
+  * **Fixed** `3376 <https://github.com/pymupdf/PyMuPDF/issues/3376>`_: PyMuPDF==1.24.1: Redacting results are not as expected in 1.24.1.
```
Comment (julian-smith-artifex-com) [inline changes.txt:31]: I don't think this was fixed in 1.24.2.

File: src/utils.py [hunk 2]
```diff
@@ -480,16 +480,11 @@ def get_text_blocks(
...
     """
     fitz.CheckParent(page)
     if flags is None:
-        flags = (
-                fitz.TEXT_PRESERVE_WHITESPACE
-                    | fitz.TEXT_PRESERVE_IMAGES
-                    | fitz.TEXT_PRESERVE_LIGATURES
-                    | fitz.TEXT_MEDIABOX_CLIP
-                )
+        flags = fitz.TEXTFLAGS_BLOCKS
```
Comment (julian-smith-artifex-com) [inline src/utils.py:487]: Could you explain this change in the commit message? It seems to be changing to TEXTFLAGS_BLOCKS=
        | TEXT_PRESERVE_LIGATURES
        | TEXT_PRESERVE_WHITESPACE
        | TEXT_MEDIABOX_CLIP
        | TEXT_CID_FOR_UNKNOWN_UNICODE
?

File: changes.txt (hunk 1 above)
Comment (JorjMcKie) [reply changes.txt:31]: correct - also found that out in the meantime ...

File: src/utils.py (hunk 2 above)
Comment (JorjMcKie) [reply src/utils.py:487]: This method is an alias for get_text("blocks") so it must use the same flags. We wouldn't be able to give a good answer when being asked why the behavior is different.

**Review by julian-smith-artifex-com — APPROVED** [review]

**Pull request merged**